plugins/@grouparoo/snowflake/src/lib/export/exportRecords.ts

```diff
@@ -254,7 +254,11 @@ function formatPayloadValues(payloadValues: string[]) {
   return payloadValues.reduce((prev, curr, idx) => {
     return (
       prev +
-      (!curr || curr.length === 0 ? null : `'${curr}'`) +
+      (curr === null ||
+      curr === undefined ||
+      curr.toString().trim().length === 0
+        ? null
+        : `'${curr.toString().replace(new RegExp("'", "gi"), "")}'`) +
```

Comment:
anything that is trying to fix u pthe string is going to be a risk in general.
Above, it's doing `${values.join(", ")` that should be something more like [this](https://docs.snowflake.com/en/user-guide/nodejs-driver-use.html#binding-array-for-bulk-insert).

```
connection.execute({
  sqlText: 'select c1 from (select ? as c1 union all select ? as c1) where c1 = ?;',
  binds: [1, 2, 1]
});
```

each record would be a `?` and then the right thing would passed to `binds`.

Reply:
ok. I'll try this out.

Reply:
It only works with `select` (we're already using it) and `insert` I changed to use it.